core/src/services/s3/backend.rs

```diff
@@ -373,8 +373,7 @@ impl S3Builder {
 
     /// Region represent the signing region of this endpoint.
     ///
-    /// - If region is set, we will take user's input first.
-    /// - If not, we will use `us-east-1` as default.
+    /// This is a required parameter.
```

Comment:
Only required for AWS S3. Other services like minio will still fallback to `us-east-1` as more of them assumed.